ports/vcpkg-msbuild/vcpkg_msbuild_install.cmake

```
@@ -0,0 +1,323 @@
+#[===[.md:
+# vcpkg_msbuild_install
+
+Build and install a msbuild-based project.
+
+```cmake
+vcpkg_msbuild_install(
+    SOURCE_PATH <source-path>
+    PROJECT_FILE <path-to-solution-or-project>
+    [TARGET <target>]
+    [INCLUDES_DIRECTORY <path-to-include-dir>]
+
+    [RELEASE_CONFIGURATION <configuration>]
+    [DEBUG_CONFIGURATION <configuration>]
+    [OPTIONS <option>...]
+    [OPTIONS_RELEASE <option>...]
+    [OPTIONS_DEBUG <option>...]
+
+    [PLATFORM_VERSION <platform-version>]
+    [PLATFORM_ARCHITECTURE <architecture>]
```

```suggestion
    [PLATFORM <msbuild platform>]
```
Platform is a technical term in MSBuild, that has special significance for the built-in VC targets.